proto/vtctldata.proto

```diff
@@ -42,6 +43,114 @@ message ExecuteVtctlCommandResponse {
   logutil.Event event = 1;
 }
 
+message ChangeTabletTypeRequest {
+  topodata.TabletAlias tablet_alias = 1;
+  topodata.TabletType db_type = 2;
+  bool dry_run = 3;
+}
+
+message ChangeTabletTypeResponse {
+  topodata.Tablet before_tablet = 1;
+  topodata.Tablet after_tablet = 2;
+  bool was_dry_run = 3;
+}
+
+message CreateKeyspaceRequest {
```

Comment:
Just curious -- [CreateKeyspace docs](https://vitess.io/docs/reference/programs/vtctl/keyspaces/#createkeyspace) don't mention a few of these params. Do you think this is usual doc drift or are there $reasons they aren't included? (Or am I looking at the wrong thing entirely?)

Reply:
Yeah, my broad answer on this is that these are similar to but distinctly **not** the vtctl commands, and that includes the CLI definitions (and, those docs don't actually list [all the flags you can actually use](https://github.com/vitessio/vitess/blob/4fea10e8718e4285f1a1c2675cb85ed458ba4565/go/vt/vtctl/vtctl.go#L1619-L1628) in the current version). So exact matching is not a requirement, and I figured this was a good opportunity to improve on the existing CLI options rather than just carry them over.